app/jobs/bulkrax/delete_file_set_job.rb

```diff
@@ -2,4 +2,22 @@
 
 module Bulkrax
   class DeleteFileSetJob < DeleteJob; end
```

Comment:
Also, ignore me if I'm reading this wrong, but is this an extra end?

Reply:
grr - I really hate the `; end` syntax. it takes an important thing and hides it for no real gain but to save a linefeed. fixed

Reply:
this was a really good catch Shana. its legal to define the method on the module so its syntaxitcally correct but would not have ever been called the way I had it